src/main/java/sirius/biz/tycho/dashboards/Chart.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.dashboards;
+
+import sirius.web.services.JSONStructuredOutput;
+
+/**
+ * Represents a chart which is generated by {@link ChartProvider#resolveChart(String, String)}.
+ */
+public interface Chart {
+
+    /**
+     * Generates the JSON representation required by the UI to render the chart.
+     *
+     * @param output the target to write the JSON to
+     */
+    void writeJSON(JSONStructuredOutput output);
```

Comment:
Would be more fitting to our general naming scheme when writeJSON would be named writeJson, wouldn't it?